src/std/os/kqueue.ss

```diff
@@ -43,44 +59,46 @@ package: std/os
   (check-ptr (make_kevents size)))
 
 (def (kevent-set! kevts ix
-      ident: ident     filter:       filter
-      flags: (flags 0) filter-flags: (filter-flags 0)
-      data:  (data 0)  user-data:    (user-data #f))
+		  ident: ident     filter:       filter
+		  flags: (flags 0) filter-flags: (filter-flags 0)
+		  data:  (data 0)  user-data:    (user-data #f))
   (ev_set kevts ix ident filter flags filter-flags data user-data))
 
-(def (kevent kqueue change-list nchanges event-list nevents timeout)
+(def (kevent kqueue change-list nchanges event-list nevents)
   (do-retry-nonblock
-   (_kevent (fd-e kqueue) change-list nchanges event-list nevents timeout)
-   (kevent kqueue change-list nchanges event-list nevents timeout)))
+   (_kevent (fd-e kqueue) change-list nchanges event-list nevents
+	    timespec-zero)
+   (kevent kqueue change-list nchanges event-list nevents)))
 
 (def (kqueue-poll kqueue events nevents)
-  (kevent kqueue #f 0 events nevents timespec-zero))
+  (kevent kqueue #f 0 events nevents))
 
-(def (kqueue-kevent-add kqueue dev filter (additional-flags 0) (filter-flags 0) (data 0))
+(def (kqueue-kevent-add kqueue dev filter (additional-flags 0)
+			(filter-flags 0) (data 0))
```

Comment:
any reason to change this line?

Reply:
Just because I don't like lines that are longer than 80 characters. I will revert it, if you wish. And it's ok if you are pedantic. Anything else?

Reply:
yeah, revert it.  other than the pedanticness this is ready to merge!